airbyte-integrations/connectors/source-fleetio/.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
+!Dockerfile
```

Comment:
Note to self: we should remove `!Dockerfile`, `!setup.py` and friends once we complete the migration to Poetry for all sources.